deeplinkdispatch/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkResult.java

```diff
@@ -0,0 +1,112 @@
+package com.airbnb.deeplinkdispatch;
+
+import android.content.Intent;
+import androidx.core.app.TaskStackBuilder;
+
+import javax.annotation.Nullable;
+
+@SuppressWarnings({"unused", "WeakerAccess"})
+public final class DeepLinkResult {
```

Comment:
imo we should really add kotlin to this project and and use kotlin for all new code - continuing with Java is tech debt.

Reply:
@elihart 

- This file is just green because I moved `DeepLinkResult` from module `deeplinkdispatch-base` to module `deeplinkdispatch`.
- Do you think this [principle from google](https://google.github.io/eng-practices/review/reviewer/standard.html) applies?

> In general, reviewers should favor approving a CL once it is in a state where it definitely improves the overall code health of the system being worked on, even if the CL isn’t perfect.